vagrant/roles/candlepin-user/tasks/main.yml

```diff
@@ -1,22 +1,67 @@
+- name: Checkout Candlepin
```

Comment:
It will only run if that is defined and set to a non-false value.  By default it is set to False in vagrant/roles/candlepin-user/defaults/main.yml (added in this PR), and when run will simply skip.